scss/_buttons.scss

```diff
@@ -116,7 +116,7 @@
   background-color: transparent;
   border: 2px dashed $gray-600;
 
-  div {
+  .icon-holder {
```

Comment:
I believe making this change is a breaking change. Most likely we need to leave the div here and then add .icon-holder as well.